NEWS.md

```diff
@@ -1,5 +1,11 @@
 # usethis (development version)
 
+* Add trailing slash to DESCRIPTION URLs created from `use_github_links()` to adhere to new CRAN URL checks
```

Comment:
Let's keep the PR more focused on one thing. I'm pretty sure we fixed our slash problems already? E.g. #1526 and #1527. Even if not, it's easier to think about these things separately.

Reply:
Sounds good, I'll revert and open another PR. I think the slash problems were only addressed in `code-of-conduct.R`, `logo.R`, `pkgdown.R`